velox/dwio/common/ColumnVisitors.h

```diff
@@ -913,10 +921,10 @@ class DictionaryColumnVisitor
           dictMask,
           reinterpret_cast<const int32_t*>(filterCache() - 3),
           indices);
-      auto unknowns = simd::toBitMask(xsimd::batch_bool<int32_t>(
-          simd::reinterpretBatch<uint32_t>((cache & (kUnknown << 24)) << 1)));
-      auto passed = simd::toBitMask(
-          xsimd::batch_bool<int32_t>(simd::reinterpretBatch<uint32_t>(cache)));
+      auto cache_bits = (simd::reinterpretBatch<uint32_t>(cache) & xsimd::batch<uint32_t>(1)) != xsimd::batch<uint32_t>(0);
```

Comment:
Can we make sure all these comparison with 0 does not result in extra instructions?